sync-github-ado.yml

```diff
@@ -0,0 +1,34 @@
+# variables:
+  # ADOUrl: defined on azure portal
```

Comment:
Consider define the azure repo URL directly here in code over a config in azure pipelines. Code is the source of truth for everything.

Reply:
Do not think it's good to expose the internal Azure DevOps repository Url in the public GitHub repo